invenio_communities/views/decorators.py

```diff
@@ -66,3 +66,59 @@ def view(**kwargs):
             raise PermissionDeniedError()
 
     return view
+
+def require_community_read(f):
+    """Decorate a view to require community read permissions for accessing the 
+    view."""
+    @wraps(f)
+    def view(**kwargs):
+        """."""
+        pid_value = kwargs['pid_value']
+        community = service().read(
```

Comment:
you can avoid reading the community again (saving a potential DB call), given that it is already resolved with `@pass_community`.